plugins/codemirror/source.html

```diff
@@ -103,7 +103,8 @@
 
 	// Borrowed from codemirror.js themeChanged function. Sets the theme's class names to the html element.
 	// Without this, the background color outside of the codemirror wrapper element remains white.
-	document.documentElement.className += CMsettings.config.theme.replace(/(^|\s)\s*/g, " cm-s-");
```

Comment:
If this is causing an error, rather than commenting out, why not just do a check on CMsettings.config.theme?